hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -841,6 +841,14 @@ public class HoodieWriteConfig extends HoodieConfig {
       .sinceVersion("1.0.0")
       .withDocumentation("Whether to enable incremental table service. So far Clustering and Compaction support incremental processing.");
 
+  public static final ConfigProperty<Boolean> STREAMING_WRITES_TO_METADATA_TABLE = ConfigProperty
+      .key("hoodie.write.streaming.writes.to.metadata")
+      .defaultValue(false)
```

Comment:
The default here is a bit misleading. Below in `getDefaultForStreamingWritesToMetadataTable` seems like default is true for Spark so is it possible to remove the default for this config?